include/SDL_config_windows.h

```diff
@@ -201,20 +210,6 @@ typedef unsigned int uintptr_t;
 #define HAVE_STDDEF_H   1
 #endif
 
-/* Check to see if we have Windows 10 build environment */
-#if defined(_MSC_VER) && (_MSC_VER >= 1911)        /* Visual Studio 15.3 */
-#include <sdkddkver.h>
-#if _WIN32_WINNT >= 0x0601  /* Windows 7 */
-#define SDL_WINDOWS7_SDK
-#endif
-#if _WIN32_WINNT >= 0x0602  /* Windows 8 */
-#define SDL_WINDOWS8_SDK
-#endif
-#if _WIN32_WINNT >= 0x0A00  /* Windows 10 */
-#define SDL_WINDOWS10_SDK
-#endif
-#endif /* _MSC_VER >= 1911 */
-
```

Comment:
It has since been removed, but we did use the SDL_WINDOWS8_SDK definition at some point. It's convenient to know which SDK is available without the _WIN32_WINNT macro sausage.

Reply:
Either way (granular `HAVE_FOO_H` vs. general `HAVE_WINDOWSFOO_SDK` defines) is fine with me. We just need to pick one and use it consistently in CMake and non-CMake, or we'll end up back in this boat again when someone adds another new feature that requires a newer SDK.

If we want to keep those `HAVE_SDK` defines, maybe we should merge #4901 instead and I can mess with the `sdkddkver.h` stuff in a separate PR.

Reply:
I'm fine with the targeted defines, and it looks like nobody else is using the more general ones, so let's go with your approach for now.